src/cslStyles.js

```diff
@@ -74,19 +74,21 @@ define(['src/urlUtils', 'src/debug'], function (CSLEDIT_urlUtils, debug) {
 		return styleTitle
 			.replace(/&/g, "and")
 			.replace(/\([A-Z]*\)/g, "") // remove upper case text (acronyms) in parentheses
+			.replace(/\([^\(]*\)$/, "")) //remove content between last set of parentheses
+			.replace(/\[[^\[]*\]$/, "")) //remove content between last set of square parentheses
```

Comment:
(this typo was fixed in https://github.com/citation-style-language/csl-editor/commit/c07fc753a9405b2930728696ff950a87881fc957)